fix: naj faq - improved 'missing-methods' section (#1139)
near/docs: docs/4.tools/near-api-js/faq.md

### Can I use `near-api-js` on a static html page? {#can-i-use-near-api-js-on-a-static-html-page}

> You can load the script form a CDN.

Reviewer: ```suggestion
You can load the script from a CDN.
```

## Common Errors {#common-errors}

### Missing contract methods {#missing-contract-method}

> When you construct a `Contract` instance on client-side, you need to specify
> the methods the contract has. If you misspell, mismatch or miss method names - you
> receive errors about missing methods.

Reviewer: ```suggestion
When constructing a `Contract` instance on the client-side, you need to specify
the contract's methods. If you misspell, mismatch, or miss method names - you'll
receive errors about missing methods.
```

There are few cases for missing or wrong methods:
- When you call a method that you didn't specify in constructor.
- When you call that doesn't exist on the contract on the blockchain (but you did specify it in the client-side constructor).
- When you mismatch between `viewMethods` and `changeMethods`.

> For example lets look at the following contract code.
> It contains one `view` and one `call` method:

Reviewer: ```suggestion
For example, let's look at the following contract code.
It contains one `view` and one `call` method:

```

> It contains one `view` and one `call` methods:

Reviewer: ```suggestion
It contains one `view` and one `call` method:
```
